data/trie/factory/trieCreator_test.go

```diff
@@ -1,3 +1,67 @@
 package factory
 
-// TODO add tests
+import (
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/config"
+	"github.com/ElrondNetwork/elrond-go/data/mock"
+	"github.com/ElrondNetwork/elrond-go/data/trie"
+	"github.com/stretchr/testify/assert"
+)
+
+func getArgs() TrieFactoryArgs {
+	return TrieFactoryArgs{
+		Marshalizer: &mock.MarshalizerMock{},
+		Hasher:      &mock.HasherMock{},
+		PathManager: &mock.PathManagerStub{},
+		ShardId:     "0",
+		Cfg: config.StorageConfig{
+			Cache: config.CacheConfig{
+				Type:   "LRU",
+				Size:   100,
+				Shards: 2,
+			},
+			DB: config.DBConfig{
```

Comment:
Maybe have a mechanism as to not use physical files for unit/integration tests.

Reply:
removed.